content/dstu2/medication-statement.md

```diff
@@ -26,3 +30,107 @@ Search for MedicationStatements that meet supplied query parameters:
 
 <%= headers 200 %>
 <%= json(:dstu2_medication_statement_bundle) %>
+
+## Create
+
+Create new medicationStatement.
+
+    POST /MedicationStatement
+
+### Headers
+
+To successfully POST a medicationStatement, the following headers must be provided. MedicationStatement creation is supported from the closed endpoint only.
+
+    Content-Type: application/json+fhir
+    Authorization: <OAuth2 Bearer Token>
+
+### Body fields
+
+<%= definition_table(:medication_statement, :create, :dstu2) %>
+
+### Example Body
+
+<%= json(:dstu2_medication_statement_create) %>
+
+### Response
+
+<%= headers 201 %>
+<pre class="terminal">
+    Connection → Keep-Alive
+    Content-Encoding → gzip
+    Content-Length → 20
+    Content-Type → text/html; charset=UTF-8
+    Date → Wed, 13 Jan 2016 21:45:47 GMT
+    Keep-Alive → timeout=15, max=100
+    Last-Modified → Tue, 15 Dec 2015 19:13:20 GMT
+    Status → 201 Created
+    access-control-allow-methods → DELETE, GET, POST, PUT, OPTIONS, HEAD
+    access-control-allow-origin → *
+    access-control-expose-headers → ETag, Content-Location, Location, X-Request-Id, WWW-Authenticate, Date
+    access-control-max-age → 0
+    cache-control → no-cache
+    etag → W/"0"
+    location → https://fhir.sandboxcernerpowerchart.com/dstu2/d075cf8b-3261-481d-97e5-ba6c48d3b41f/MedicationStatement/20465903
+    server-response-time → 1260.984596
+    strict-transport-security → max-age=631152000
+    vary → Origin,User-Agent,Accept-Encoding
+    x-content-type-options → nosniff
+    x-frame-options → SAMEORIGIN
+    x-request-id → 682c633c-b20f-4f6f-8fae-c58b3aeffe04
+    x-runtime → 1.260940
+    x-xss-protection → 1; mode=block
+</pre>
+
+The `etag` response header indicates the current `If-Match` version to use on subsequent updates.
+
+## Update
+
+Update an existing medicationStatement.
+
+    PUT /MedicationStatement
+
+Note that any field which is missing will be interpreted as nulling out or removing data from the resource. See [FHIR Update](http://hl7.org/fhir/DSTU2/http.html#update) for additional details about update operations.
+
+### Headers
+
+To successfully PUT an medicationStatement, the following headers must be provided. MedicationStatement updates are supported from the closed endpoint only.
```

Comment:
Watch out for the articles too, "an medicationStatement" => "a MedicationStatement"